app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb

```diff
@@ -6,8 +6,8 @@
 
 <div class="content">
   <section class="col col-720">
-    <p>Bursaries are tax-free payments you receive if you're training to teach certain subjects. If you're eligible for a bursary, you'll get it automatically.</p>
-    <p>Scholarships are awarded by organisations with a passion for their subject area. They combine tax-free payments with extra benefits like subject-specific support, mentoring and networking opportunities. If you're eligible, you can apply directly to the scholarship provider.</p>
+    <p>Postgraduate bursaries are tax-free payments you receive if you're training to teach certain subjects. You do not need to apply for a bursary, you'll automatically get one if you're eligible.</p>
```

Comment:
```suggestion
    <p>Postgraduate bursaries are tax-free payments you receive if you're training to teach certain subjects. You do not need to apply for a bursary, you'll automatically get one if you're eligible.</p>
```
This is very pedantic - feels like this should be a bigger pause than a comma (but obviously not a semi-colon!).  Feel free to ignore this though!